Algorithm.CSharp/OptionIndicatorsRegressionAlgorithm.cs

```diff
@@ -61,17 +66,33 @@ public override void OnData(Slice slice)
 
                 _delta.Update(underlyingDataPoint);
                 _delta.Update(optionDataPoint);
+
+                _gamma.Update(underlyingDataPoint);
+                _gamma.Update(optionDataPoint);
+
+                _vega.Update(underlyingDataPoint);
+                _vega.Update(optionDataPoint);
+
+                _theta.Update(underlyingDataPoint);
+                _theta.Update(optionDataPoint);
+
+                _rho.Update(underlyingDataPoint);
+                _rho.Update(optionDataPoint);
```

Comment:
Whops! If using the helper methods, none of these Update calls should be required, the whole OnData really. I gave it a quick test locally and it's failing❗, requires some debugging